### PR TITLE
Add group badge to connect interface and collaborator dropdown

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

**Comment by broskoski** [comment]
Closes #1668
Closes #1671

File: components/connect/components/item/index.jade [hunk 1]
```diff
@@ -1,13 +1,16 @@
 - isGroup = channel.group && channel.group.id
 - identifiable = isGroup ? channel.group : channel.user
 - name = isGroup ? identifiable.name : identifiable.username
-
+-
```
Comment (dzucconi) [inline components/connect/components/item/index.jade]: what's this dash?

Comment (broskoski) [reply components/connect/components/item/index.jade]: Ah weird typo, will remove

File: react/pages/feed/components/EmptyConnectTwitter/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ import EmptyFeedConnectTwitterQuery from 'react/pages/feed/components/EmptyConne
 import emptyFeedConnectTwitterFragment from 'react/pages/feed/components/EmptyConnectTwitter/fragments/emptyFeedConnectTwitter';
 
 const { data: { API_URL, APP_URL } } = sharify;
-const TWITTER_AUTHENTICATION_URL = `${API_URL.replace('/v2', '')}/auth/twitter?origin=${APP_URL}/feed/find-friends`;
+const TWITTER_AUTHENTICATION_URL = `${(API_URL && API_URL.replace('/v2', ''))}/auth/twitter?origin=${APP_URL}/feed/find-friends`;
```
Comment (dzucconi) [inline react/pages/feed/components/EmptyConnectTwitter/index.js]: This is actually an issue?

Comment (broskoski) [reply react/pages/feed/components/EmptyConnectTwitter/index.js]: Yeah, just in storybooks.

Comment (dzucconi) [reply react/pages/feed/components/EmptyConnectTwitter/index.js]: Should have just hardcoded this into its own string, tbh

Comment (dzucconi) [reply react/pages/feed/components/EmptyConnectTwitter/index.js]: 👍

**Pull request merged**